tests/test_graph_components/adders/test_nlu_prediction_to_history_adder.py

```diff
@@ -0,0 +1,68 @@
+from rasa.shared.core.events import UserUttered
+from rasa.shared.nlu.constants import ENTITIES, INTENT, TEXT
+from rasa.core.channels.channel import UserMessage
+import pytest
+import freezegun
+from typing import Text, List
+from rasa.shared.core.trackers import DialogueStateTracker
+from rasa.shared.nlu.training_data.message import Message
+from rasa.engine.graph import ExecutionContext
+from rasa.engine.storage.resource import Resource
+from rasa.engine.storage.storage import ModelStorage
+from rasa.shared.core.domain import Domain
+from rasa.graph_components.adders.nlu_prediction_to_history_adder import (
+    NLUPredictionToHistoryAdder,
+)
+
+
+@pytest.mark.parametrize(
+    "messages,expected,input_channel",
+    [
+        ([], [], "slack"),
+        (
+            [
+                Message(
+                    {TEXT: "message 1", INTENT: {}, ENTITIES: [{}], "message_id": "1",}
+                ),
+                Message(
```

Comment:
Based on the comment from @wochinge that `List[Message]` can have 1 object max, should the test be updated to reflect this, i.e. remove 2nd message?

Reply:
Makes sense, also would want to drop the `metadata` one in light of the fact that isn't propagated through the `Message` any more

Reply:
Yup,also it would be a good idea to add metadata in your test when you do the `UserMessage` instantiation in L56?